spec/fixtures/emails/rfc2822/example14.eml

```diff
@@ -3,9 +3,8 @@ Date: Fri, 19 Aug 2011 10:47:17 +0900
 From: Atsushi Yoshida <atsushi@example.com>
 Reply-To: rudeboyjet@gmail.com
 Subject: Re: TEST
- 
-	=?ISO-2022-JP?B?GyRCJUYlOSVIGyhC?=
-  =?ISO-2022-JP?B?GyRCJUYlOSVIGyhC?=
+ =?ISO-2022-JP?B?GyRCJUYlOSVIGyhC?=
+ =?ISO-2022-JP?B?GyRCJUYlOSVIGyhC?=
```

Comment:
That change appears to be unrelated to this PR, though. Would limit scope to just the change at hand.